### ISSUE TITLE
Javascript FlipStrategy - Problem with CDATA Code

### ISSUE THREAD

**Issue by marcovasapollo** [ff4j/ff4j#374]
Hello and thanks for your amazing Work with ff4j!

I built a FlipStrategy wich consent to write check rules using Javascript (through rhino).

So my FF4J xml looks like this

<img width="664" alt="ff4j" src="https://user-images.githubusercontent.com/5588727/64075114-57291300-ccb4-11e9-80df-f01bc1b9c7db.png">

Of course, because of Javascript code can contain special characters like & or < or whatever, it is needed to encapsulate the code inside the a CDATA container.

Now the problem is that the ff4j parser doesn't read the cdata content properly.
I debugged the suite and discovered that the parser totally jumps that part.

<img width="718" alt="debug" src="https://user-images.githubusercontent.com/5588727/64075159-0b2a9e00-ccb5-11e9-8814-2de41b6c75dd.png">

During debug session I discovered that the getNodeValue() method is not enough to give back all the entire node content.

Is there a way to do that, maybe a minor release? 
Or you suggest to fix in another way?

Thanks in advance.

**Comment by marcovasapollo** [comment]
[SOLVED]

The `XMLParser `class works well, but it looks like that the `init(String featureName, Map<String, String> initParam)` of the `FlippingStrategy` class is called several times during the xml parsing.  
The FIRST time it is called with the correct map values, while other times it is called with "cutted" values. So, in my FlipStrategy I created the field `private static final Map<String, String> PARAMETERS = new HashMap<>();`

And I fill like this in the `init` method:

![image](https://user-images.githubusercontent.com/5588727/64076535-6a44de80-ccc6-11e9-8e26-91e00e45237c.png)

I also created copies of the data (`new String(key)` and `new String(value))` to be extremely sure to avoid posthumous manipulation.

Then, in my `evaluate` method, I use the `PARAMETERS` field instead of `initParams`

This is the way the FF4J object is initialized:

![image](https://user-images.githubusercontent.com/5588727/64076643-990f8480-ccc7-11e9-8535-f38e0f78df47.png)

FF4J version i 1.8.1

Cheers!
